src/index.js

```diff
@@ -1,3 +1,173 @@
-console.log(data);
+document.addEventListener("DOMContentLoaded", function() {
 
-// WRITE YOUR CODE BELOW!
+    const data = [
```

Comment:
why did you copy the dogs data here? you can import the data from `data.js`

Reply:
I removed it. I initially copied and pasted to just stop going back and fourth between the files :)